### PR TITLE
NOBUG: Updating readmes for cms and staging

### PR DESCRIPTION
**Pull request by danieltruong**
### Description:
Updating readmes for cms and staging.

### CODE REVIEW

File: src/cms/README.md [hunk 1]
```diff
@@ -6,12 +6,20 @@ Strapi allows administration of site data.
 Run a postgres instance
 eg: docker run --name postgres-docker -e POSTGRES_PASSWORD=postgres -p 5432:5432 -d postgres
 
-## Dev steps to run locally
+## Dev steps to initialize locally
 
 1. Install and activate NodeJS 14.
-2. Install and start up postgres locally. You'll need to create a user and database,
-   and set ``DATABASE_HOST``, ``DATABASE_PORT``, ``DATABASE_NAME``, ``DATABASE_USERNAME``,
+2. Install and start up postgres locally. You'll need to create a user and database
```
Comment (BcGovNeal) [inline src/cms/README.md:12]: Step 2 isn't necessary if you run the postgres container with `POSTGRES_DB=cms` env

E.g., `docker run --name postgres-docker -e POSTGRES_PASSWORD=postgres -p 5432:5432 -e=POSTGRES_DB=cms -d postgres`

File: src/staging/README.md [hunk 2]
```diff
@@ -1,99 +1,12 @@
...
+1. Make sure Strapi is running. Follow the steps found in ``src/cms/``
+2. Copy ``.env.example`` as ``.env``.
+  * ```REACT_APP_CMS_BASE_URL=http://localhost:1337``` must be set. Be sure your Strapi server is actually running on port ``1337``.
 
-_Have another more specific idea? You may want to check out our vibrant collection of [official and community-created starters](https://www.gatsbyjs.com/docs/gatsby-starters/)._
+## Running local dev server
 
-## 🚀 Quick start
+1. ``yarn``
+2. ``yarn start``
```
Comment (cole) [inline src/staging/README.md:10]: Maybe note the develop command as well for reloading?

**Review by cole — APPROVED** [review]
Looks good with two minor notes, thanks for updating 🙌

File: src/cms/README.md (hunk 1 above)
Comment (danieltruong) [reply src/cms/README.md]: [c58f1c3](https://github.com/bcgov/bcparks.ca/pull/258/commits/c58f1c37ae15505a413ecb03b7ab69bcb3e51529)

**Review by BcGovNeal — APPROVED** [review]

File: src/staging/README.md (hunk 2 above)
Comment (BcGovNeal) [reply src/staging/README.md:10]: The start command runs the develop command

Comment (danieltruong) [reply src/staging/README.md:10]:
```  "scripts": {
    "build": "gatsby build",
    "develop": "gatsby develop -H 0.0.0.0",
    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
    "start": "npm run develop",
    "serve": "gatsby serve",
    "clean": "gatsby clean",
    "test": "jest"
  },```

Start should run develop which runs gatsby develop -H 0.0.0.0

Comment (cole) [reply src/staging/README.md:10]: Ah, my mistake, got it mixed up with serve. Start is fine then.

Bit odd though to call npm via npm or yarn.. 🧐

**Pull request merged**